pkg/apis/pubsub/v1alpha1/pull_subscription_types.go

```diff
@@ -59,6 +59,9 @@ var _ = duck.VerifyType(&PullSubscription{}, &duckv1.Conditions{})
 
 // PullSubscriptionSpec defines the desired state of the PullSubscription.
 type PullSubscriptionSpec struct {
+	// This brings in CloudEventOverrides and Sink.
+	duckv1.SourceSpec
```

Comment:
I think this needs a json inline tag here

Reply:
Done.